shared/searchv3/services-filter/index.js

```diff
@@ -109,7 +118,7 @@ class Filter extends Component<void, Props, {hoveredService: ?Constants.Service}
             service={service}
             selected={service === this.props.selectedService}
             hovering={service === this.state.hoveredService}
-            onHover={this._hoverChanged}
+            onHover={isHovering => this._hoverChanged(service, isHovering)}
```

Comment:
why did you remove the service and make this closure? we want to avoid having closures as props like this